### ISSUE TITLE
app.yaml in main directory not found in version 1.2.2

### ISSUE THREAD

**Issue by Jipje** [google-github-actions/deploy-appengine#311]
### TL;DR

I receive `Error: google-github-actions/deploy-appengine failed with: could not find an appyaml in [app.yaml]` when running `google-github-actions/deploy-appengine@v1`. Specifying previous version `google-github-actions/deploy-appengine@v1.2.1` fixes the issue.

### Expected behavior

I expect the app.yaml in the main directory of the project to be found. This works on version `@v0` as well as `@v1.2.1` but is broken in the latest release it seems.

### Observed behavior

When the deploy is ran I receive the following error: `Error: google-github-actions/deploy-appengine failed with: could not find an appyaml in [app.yaml]`

I tried explicitly pointing towards the correct working directory and specifying the `app.yaml` filename using the `working_directory` and `deliverables` tags. I could not get that to work (I guess that's on me) but I expected reverse compatibility with previous versions I guess.

### Action YAML

```yaml
name: Deploy to Google App Engine

on:
  push:
    branches:
      - "main"

jobs:
  deploy_to_app_engine:
    name: deploy to google app engine
    runs-on: ubuntu-latest

    permissions:
      contents: 'read'
      id-token: 'write'

    steps:
      - uses: 'actions/checkout@v3'

      - id: 'auth'
        name: 'Authenticate to Google Cloud'
        uses: 'google-github-actions/auth@v1'
        with:
          workload_identity_provider: 'my-wif-provider'
          service_account: 'service-account@project-name.iam.gserviceaccount.com'

      - id: 'deploy'
        name: 'Deploy platform to Google Cloud - App Engine'
        uses: 'google-github-actions/deploy-appengine@v1.2.1'
        with:
          env_vars: |-
            TEST_MYSQL_PASSWORD=${{ secrets.TEST_MYSQL_PASSWORD }}
            MYSQL_PASSWORD=${{ secrets.MYSQL_PASSWORD }}
```

### Log output

```text
Run google-github-actions/deploy-appengine@v1
Error: google-github-actions/deploy-appengine failed with: could not find an appyaml in [app.yaml]
```

### Additional information

Above log output is received when running the offered action.yaml with `@v1` instead of `@v1.2.1` as is described there right now.

**Comment by sethvargo** [comment]
Hi @Jipje - what does your `app.yaml` look like?

**Comment by Jipje** [comment]
This is what our `app.yaml` looke like:

```
runtime: python310

instance_class: F1

env_variables:
  SQLALCHEMY_WARN_20: True
  PYTHONWARNINGS: "always::DeprecationWarning"
```
